common/validator/queryValidator_test.go

```diff
@@ -18,11 +18,12 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package common
```

Comment:
suggestion: this should belong to `common/elasticSearch/validator` not `common/validator/`

Reply:
moved